docs/configuration.md

```diff
@@ -364,6 +376,18 @@ Permitted values are `error`, `warn`, `info` and `debug`. The default is
         region: fr
         container: containername
         rootdirectory: /swift/object/name/prefix
+      oss:
+        accesskeyid: accesskeyid
+        accesskeysecret: accesskeysecret
+        region: OSS region name
+        endpoint: optional endpoints
+        internal: optional internal endpoint
+        bucket: OSS bucket
+        encrypt: optional data encryption setting
+        secure: optional ssl setting
+        chunksize: optional size valye
+        rootdirectory: optional root directory
+      inmemory:
```

Comment:
Hmm I see the comment in the source. GitHub seems to be hiding it in this diff.  You can see it on the page here:

https://github.com/moxiegirl/distribution/blob/fix-issue-1254/docs/configuration.md

It also shows fine in the built docs:

![image](https://cloud.githubusercontent.com/assets/1347209/12490038/5ebb6f9c-c028-11e5-9eb6-a98492a9dca3.png)